ext/CMakeLists.txt

```diff
@@ -62,6 +59,27 @@ if(TRACY_ENABLE)
     endif()
 endif(TRACY_ENABLE)
 
+# NOTE: We are not using the latest versions of spdlog, since the
+#     : version of fmt it is bundled with deprecated some arg mapping
+#     : that we use in a lot of places (formatting enums as ints and
+#     : formatting non-void pointers).
+CPMAddPackage(
+    NAME spdlog
+    GITHUB_REPOSITORY gabime/spdlog
+    GIT_TAG v1.9.2
+) # defines: spdlo
```

Comment:
Spelling: # defines: spdlog

It might be better if you just cherry pick my upstream changes here: <https://github.com/LandSandBoat/server/commit/8d952f35679889f8d46b9c7ee87e1379f290224a>